docs/integration.rst

```diff
@@ -271,6 +271,23 @@ AzureContainerVolumeHook
 
 .. _AWS:
 
+Azure Kubernetes Service
+'''''''''''''''''''''''''
+Azure Kubernetes Service (AKS) simplifies the deployment and operations of Kubernetes and enables you to dynamically scale your application infrastructure.
+The AzureKubernetesServiceHook requires a service principal. The credentials for this principal can either be defined in the extra field `key_path`, as an 
+environment variable named `AZURE_AUTH_LOCATION`.
+
+- :ref:`AzureKubernetesOperator` : Start new AKS.
+- :ref:`AzureKubernetesServiceHook` : Wrapper around a aks.
+
+AzureKubernetesOperator
+"""""""""""""""""""""""""""""""
+ .. autoclass:: airflow.contrib.operators.aks_operator.AzureKubernetesOperator
+
```

Comment:
Missing `no-index` flag.

Reply:
@mik-laj  what is no-index flag? I didnt find it in documentation

Reply:
Now it does not matter anymore. The structure of the integration.rst file has changed. Look how this file looks like now

if you have an autoclass directive in the file, you can refer to it by reference: class:.  If you have an autoclass directive in two files then you need to decide which is more important and to which links should be addressed.